package.json

```diff
@@ -26,16 +26,22 @@
     "ansi": "^0.3.1",
     "bplist-parser": "^0.1.0",
     "cross-spawn": "^6.0.5",
+    "deep-freeze": "^0.0.1",
+    "detect-indent": "^5.0.0",
+    "detect-newline": "^2.1.0",
     "elementtree": "0.1.7",
     "endent": "^1.1.1",
+    "fast-deep-equal": "^2.0.1",
     "fs-extra": "^7.0.0",
     "glob": "^7.1.2",
     "minimatch": "^3.0.0",
     "plist": "^3.0.1",
     "q": "^1.4.1",
+    "stringify-package": "^1.0.0",
```

Comment:
this package is [deprecated](https://www.npmjs.com/package/stringify-package), the replacement is https://www.npmjs.com/package/@npmcli/package-json